features/ouders.feature

```diff
@@ -2,113 +2,349 @@
+    Dan heeft de ouder met ouderAanduiding 'ouder1' de volgende gegevens
+    | naam                                          | waarde     |
+    | type                                          | Ouder      |
+    | ouderAanduiding                               | ouder1     |
+    | datumIngangFamilierechtelijkeBetrekking.datum | 2019-03-17 |
+    En heeft de ouder met ouderAanduiding 'ouder1' de volgende 'naam' gegevens
+    | naam      | waarde           |
+    | voornamen | Olivia Ghislaine |
+    En heeft de ouder met ouderAanduiding 'ouder2' de volgende gegevens
+    | naam                                          | waarde     |
+    | type                                          | Ouder      |
+    | ouderAanduiding                               | ouder2     |
+    | datumIngangFamilierechtelijkeBetrekking.datum | 2019-03-17 |
+    En heeft de ouder met ouderAanduiding 'ouder2' de volgende 'naam' gegevens
+    | naam      | waarde    |
+    | voornamen | Kevin     |
+
+  @gba
+  Scenario: ontkenning gevolgd door erkenning
+    Gegeven de persoon met burgerservienummer 999996162 heeft de volgende ouders in de registratie
+    | Categorie | Voornamen (02.10)  | Onjuist (84.10) |
+    | 2         | Chantal            |                 |
+    | 3         | Wieger             |                 |
+    | 53        |                    |                 |
+    | 53        | Mark               | O               |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                                              |
+    | type                | RaadpleegMetBurgerservicenummer                     |
+    | burgerservicenummer | 999996162                                           |
+    | fields              | ouders.ouderAanduiding,ouders.naam.voornamen,ouders |
+    Dan heeft de ouder met ouderAanduiding 'ouder1' de volgende gegevens
+    | naam            | waarde |
+    | type            | Ouder  |
+    | ouderAanduiding | ouder1 |
+    En heeft de ouder met ouderAanduiding 'ouder1' de volgende 'naam' gegevens
+    | naam      | waarde  |
+    | voornamen | Chantal |
+    En heeft de ouder met ouderAanduiding 'ouder2' de volgende gegevens
+    | naam            | waarde |
+    | type            | Ouder  |
+    | ouderAanduiding | ouder2 |
+    En heeft de ouder met ouderAanduiding 'ouder2' de volgende 'naam' gegevens
+    | naam      | waarde    |
+    | voornamen | Wieger    |
+
+@gba
 Rule: de geleverde oudergegevens zijn de gegevens zoals die staan op de persoonslijst van de gevraagde persoon
-  # bij het raadplegen van een persoon worden alleen gegevens uit de persoonslijst van de gevraagde persoon gebruikt, en nooit gegevens van de persoonslijst van de ouder
+  # Bij het raadplegen van een persoon worden alleen gegevens uit de persoonslijst van de gevraagde persoon gebruikt, en nooit gegevens van de persoonslijst van de ouder
 
+  @gba
```

Comment:
Zomaar een vraagje: Op het moment dat mijn vader zijn geslacht laat wijzigen wordt mij dan gevraagd of ik dat erken?